Arduboy.cpp

```diff
@@ -205,8 +196,7 @@ void Arduboy::drawCircle(int16_t x0, int16_t y0, int16_t r, uint8_t color)
   }
 }
 
-void Arduboy::drawCircleHelper
-(int16_t x0, int16_t y0, int16_t r, uint8_t cornername, uint8_t color)
+void Arduboy::drawCircleHelper(int16_t x0, int16_t y0, int16_t r, uint8_t cornername, uint8_t color)
```

Comment:
changing function declaration style from here down for all functions changed. Make sure to keep the original formatting in your documents.